docs/core/integration-quality-scale/rules/iqs046.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "IQS046 - The documentation provides a list of known supported / unsupported devices"
+---
+
+IQS046 is as follows:
+
+The documentation provides a list of known supported / unsupported devices
+
+## Reasoning
+
+A lot of Home Assistant users buy devices based on if Home Assistant supports them.
+To make it easier for users to find out if a device is supported, the documentation should provide a list of known supported and unsupported devices.
```

Comment:
I don't think we need to list both supported and unsupported devices unless the situation requires that. It may be possible to assume that all devices from a manufacturer are supported except the listed unsupported ones and vice versa. Each integration can decide what is needed to describe the situation.

Listing explicitly all devices from a manufacturer if all are known to be supported doesn't seem needed. A sentence saying that all devices are supported would be enough, I think.
```suggestion
To make it easier for users to find out if a device is supported, the documentation should provide a list of known supported or unsupported devices.
```

Reply:
That is a good point, but I do think that having some sort of consistency would be nice. It would also increase confidence of users if they find that their device is supported, where as of now, it feels more like a gamble. (and I think it would be required to have some sort of disclaimer telling that this can't be 100% accurate)

Reply:
I don't think we always need to have a header for supported devices and a header for unsupported devices. That would just become boilerplate for many integrations. It's enough that we say that integrations should document this.